src/Native/build.proj

```diff
@@ -68,19 +72,29 @@
       <PackageRid>$(PackageRid)-$(TargetArchitecture)</PackageRid>
     </PropertyGroup>
 
-    <ItemGroup>
+    <ItemGroup Condition="'$(UseIntrinsics)' != 'true'">
       <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)CpuMathNative$(NativeLibExtension)"
-                          RelativePath="Microsoft.ML\runtimes\$(PackageRid)\native" />
+                          RelativePath="Microsoft.ML.CpuMath\runtimes\$(PackageRid)\native" />
+    </ItemGroup>
+    <ItemGroup Condition="'$(UseIntrinsics)' == 'true'">
+      <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)CpuMathNative$(NativeLibExtension)"
+                          RelativePath="Microsoft.ML.CpuMath\runtimes\$(PackageRid)\lib\netstandard2.0" />
```

Comment:
Don't put native files under lib.  .NETCore will not like this.  For native libraries that must be constrained by both tfm and RID use [nativeassets](https://github.com/NuGet/Home/issues/2782).

Reply:
Yeah, I poked the [doc issue](https://github.com/NuGet/docs.microsoft.com-nuget/issues/238) on it.